quark/subproject.py

```diff
@@ -384,7 +406,7 @@ def same_checkout(self, other):
     def checkout(self):
         fork(['svn', 'checkout', self.url.geturl(), self.directory])
 
-    def update(self):
+    def update(self, clean=False):
```

Comment:
Maybe in the SVN case, where `clean` is not supported, it may be good to throw an error if there are local modifications, as it's unexpected to have half the dependencies clean (as it has been explicitly requested) and the other half left dirty.